rosdep/python.yaml

```diff
@@ -1606,7 +1606,10 @@ python-falcon:
   debian: [python-falcon]
   fedora: [python-falcon]
   gentoo: [dev-python/falcon]
-  ubuntu: [python-falcon]
+  ubuntu:
+    '*': [python-falcon]
+    bionic_python3: [python3-falcon]
```

Comment:
This syntax is long deprecated and should not be used.

Reply:
Thanks for the heads-up! I've adapted the PR accordingly.